assets/src/scripts/components/loadingIndicator.js

```diff
@@ -0,0 +1,7 @@
+export const loadingIndicator = function(elem, {showLoadingIndicator, sizeClass}) {
```

Comment:
Let's change the name of this module to loading-indictor.js to follow the js module naming convention already established. Same with the spec file.